crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -143,21 +147,60 @@ pub(crate) fn create(
     // Create a `.gitignore` file to ignore all files in the venv.
     fs::write(location.join(".gitignore"), "*")?;
 
+    let executable_target = if upgradeable && interpreter.is_standalone() {
```

Comment:
If this is an upgradeable request, we'll try to use a Python minor version link when creating the virtual environment (creating the symlink directory/junction if it doesn't exist).